### PR TITLE
Minor parser changes

### PR DESCRIPTION
**Pull request by tobim**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🐞 The example configuration file contained an erroneously named 'vast'
+  section. It has been changed to the correct name ('system').
```
Comment (mavam) [inline CHANGELOG.md]:
```markdown
- 🐞 The example configuration file contained an invalid section `vast`.
   This has been changed to the correct name `system`.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🐞 The example configuration file contained an invalid section `vast`.
+   This has been changed to the correct name `system`.
```
Comment (mavam) [inline CHANGELOG.md]: Please add ` [#706](https://github.com/tenzir/vast/pull/706)` to both entries.

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,14 @@ Every entry has a category for which we use the following visual abbreviations:
...
+
+- 🔄 Two minor modifications were done in the parsing framework:
+  - The parsers for enums and records now allow trailing separators.
+  - The dash ('-') was removed from the allowed characters of schema type
+    names.
```
Comment (mavam) [inline CHANGELOG.md]: The nesting doesn't look appealing to me when inspecting the log. Let's use plain paragraph style:

```markdown
- 🔄 Two minor modifications were done in the parsing framework: (i) the
  parsers for enums and records now allow trailing separators, and (ii) the
  dash (`-`) was removed from the allowed characters of schema type names.
  [#706](https://github.com/tenzir/vast/pull/706
```

File: CHANGELOG.md (hunk 2 above)
Comment (tobim) [reply CHANGELOG.md]: I opted to to link to #705 for the config file fix, since that PR contains the actual change.

**Pull request merged**